### ISSUE TITLE
router roles not showing in list view

### ISSUE THREAD

**Issue by qrkourier** [openziti/ziti-console#174]
I'm using ZAC: 2.6.6 w/ controller v0.27.5 and the router's roles don't show in the list view of routers. The roles column says "None".

**Comment by qrkourier** [comment]
This problem still exists in ZAC: 2.9.0. The router has role "public-routers," but "None" is shown in the roles column in Ziti Console.

![image](https://github.com/openziti/ziti-console/assets/1434400/9e50f63c-9b61-4429-8382-2a37711bcda7)

```bash
❯ ziti edge list ers
╭────────────┬─────────────────┬────────┬───────────────┬──────┬────────────────╮
│ ID         │ NAME            │ ONLINE │ ALLOW TRANSIT │ COST │ ATTRIBUTES     │
├────────────┼─────────────────┼────────┼───────────────┼──────┼────────────────┤
│ d6ga2DQJjp │ miniziti-router │ true   │ true          │    0 │ public-routers │
╰────────────┴─────────────────┴────────┴───────────────┴──────┴────────────────╯
results: 1-1 of 1
```

**Comment by JeremyTellier** [comment]
Fixed in 2.9.4
